docs/cli-spec.md

```diff
@@ -0,0 +1,47 @@
+# Signing CLI specficiation
+
+### Inputs
+
+The parameters to the signing client are as follows:
+
+`sign code AzureKeyVault [options] <files>`
+
+Options:
+
+- `-o | --output` - The output file. If omitted, overwrites input. Must be a directory if multiple files are specified.
+- `-b | --base-directory` - Base directory for files to override the working directory.
+- `-f | --force` - Overwrites a signature if it exists.
```

Comment:
If I understood correctly then we can sign same nupkg with multiple certificates, for example author cert, repo cert etc. Could we still do it here? 
With this option flag what exactly happens here?

Reply:
This simply replaces the signature with a new one. It does not append.